packages/zowe-explorer/src/job/init.ts

```diff
@@ -9,29 +9,36 @@
  *
  */
 
-import * as globals from "../globals";
 import * as vscode from "vscode";
 import * as jobActions from "./actions";
 import * as refreshActions from "../shared/refresh";
-import { IZoweJobTreeNode, IZoweTreeNode } from "@zowe/zowe-explorer-api";
+import { IZoweJobTreeNode, IZoweTreeNode, ZoweScheme } from "@zowe/zowe-explorer-api";
 import { Profiles } from "../Profiles";
 import { ZosJobsProvider, createJobsTree } from "./ZosJobsProvider";
 import * as contextuals from "../shared/context";
 import { ZoweJobNode } from "./ZoweJobNode";
 import { getSelectedNodeList } from "../shared/utils";
 import { initSubscribers } from "../shared/init";
 import { ZoweLogger } from "../utils/ZoweLogger";
+import { JobFSProvider } from "./JobFSProvider";
+import * as globals from "../globals";
+import { PollDecorator } from "../utils/DecorationProviders";
 
 export async function initJobsProvider(context: vscode.ExtensionContext): Promise<ZosJobsProvider> {
     ZoweLogger.trace("job.init.initJobsProvider called.");
+
+    context.subscriptions.push(vscode.workspace.registerFileSystemProvider(ZoweScheme.Jobs, JobFSProvider.instance, { isCaseSensitive: false }));
+    context.subscriptions.push(
+        vscode.commands.registerCommand("zowe.onJobChanged", (): vscode.Event<vscode.FileChangeEvent[]> => JobFSProvider.instance.onDidChangeFile)
+    );
+
     const jobsProvider = await createJobsTree(globals.LOG);
     if (jobsProvider == null) {
         return null;
     }
 
-    context.subscriptions.push(
-        vscode.commands.registerCommand("zowe.jobs.zosJobsOpenspool", (session, spoolNode) => jobActions.getSpoolContent(session, spoolNode))
```

Comment:
Do we need to notify end-users that this command went away? (i.e. add this to the ZE changelog)

Reply:
Yep, sorry again about these missing changelog entries - I should have been more diligent with that.
Resolved in upcoming commit 🙂